src/resources/js/v2/tickets-commerce.js

```diff
@@ -131,12 +169,15 @@ tribe.tickets.commerce = {};
 	 * @return {void}
 	 */
 	obj.bindCheckoutEvents = function( $container ) {
-		$document.trigger( 'beforeSetup.tribeTicketsCommerceCheckout', [ $container ] );
+		$document.trigger( 'beforeSetup.TicketsCommerceCheckout', [ $container ] );
 
 		// Bind container based events.
 		obj.bindCheckoutItemDescriptionToggle( $container );
 
-		$document.trigger( 'afterSetup.tribeTicketsCommerceCheckout', [ $container ] );
+		// Bind loader visibility.
+		obj.bindLoaderEvents();
+
+		$document.trigger( 'afterSetup.TicketsCommerceCheckout', [ $container ] );
```

Comment:
```suggestion
		$document.trigger( 'afterSetup.tecTicketsCommerce', [ $container ] );
```

Reply:
When we change these particular filters we modify `commerce/gateway/paypal/checkout.js` that make usage of these.